sirius_control/sirius_control/sirius_status.py

```diff
@@ -0,0 +1,83 @@
+import rclpy
+from rclpy.node import Node
+from industrial_msgs.msg import RobotStatus, TriState, RobotMode
+from std_msgs.msg import String
+
+
+class SiriusStatus(Node):
+
+    def __init__(self):
+        super().__init__('sirius_status')
+        self.send_status = '/sirius_status'
+        self.receive_topics = '/topic_received'
+        self.initialize_robot_status()
+        self.create_subscriptions()
+        self.publisher = self.create_publisher(RobotStatus, self.send_status, 10)
+        
+    def initialize_robot_status(self):
+        self.robot_status = RobotStatus()
+        self.robot_status.e_stopped.val = TriState.FALSE
+        self.robot_status.drives_powered.val = TriState.TRUE
+        self.robot_status.motion_possible.val= TriState.TRUE
+        self.robot_status.in_motion.val = TriState.FALSE
+        self.robot_status.in_error.val = TriState.FALSE
+        self.robot_status.mode.val = RobotMode.UNKNOWN
+        self.robot_status.error_codes = []
+
+
+    def joy_multiplexer_callback(self, msg):
+        self.get_logger().debug('Received message: "%s"' % msg.data)
```

Comment:
```suggestion
```
I still believe this log message should be removed (even if it's set to be a debug message). There is enough of ROS tools for debugging topics:
- `rqt_graph` to verify that nodes publish/subscribe to the same topic
- `ros2 topic info --verbose` - Lists all publishers and subscribers for specific topic with their details
- `ros2 topic hz` - Shows frequency of publishing messages
- `ros2 node info` - Lists publisher/subscribed topics for specific node
- `ros2 wtf` - Will warn If there are publishers without subscribers